conf/puppet.conf

```diff
@@ -1,9 +1,6 @@
-[master]
-    # Where Puppet stores dynamic and growing data.
-    vardir=/opt/puppetlabs/puppet/cache
-
-    # The Puppet log directory.
-    logdir=/var/log/puppetlabs
-
-    # Where Puppet PID files are kept.
-    rundir=/var/run/puppetlabs
+# This file can be used to override the default puppet settings.
+# See the following links for more details on what settings are available:
+# - https://docs.puppetlabs.com/references/latest/configuration.html
```

Comment:
I'd change the order of the links. I think going:

* important_settings
* about_settings
* file_main
* configuration.html

strikes a good balance between conceptual clarity and not wasting people's time. No matter what, configuration.html should be last, because it's impossible to make a list of ~230 settings welcoming and usable.